tests/rest/media/v1/test_media_storage.py

```diff
@@ -362,3 +362,12 @@ def _test_thumbnail(self, method, expected_body, expected_found):
                     "error": "Not found [b'example.com', b'12345']",
                 },
             )
+
+    def test_x_robots_tag_header(self):
```

Comment:
```suggestion
    def test_x_robots_tag_header(self):
    """
    Tests that the `X-Robots-Tag` is present, which informs web crawlers to not index, archive, or
    follow links in media.
    """
```

I'd just add this comment in again to give some explanation about why we want to see this.